torch/csrc/api/include/torch/nn/options/normalization.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <torch/arg.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <torch/types.h>
+
+namespace torch {
+namespace nn {
+
+/// Options for the `Normalize` module.
```

Comment:
nit:
```suggestion
/// Options for the `normalize` functional.
```

Reply:
Makes perfect sense this is a function not a class